### PR TITLE
Update Pull Request Template

### PR DESCRIPTION
**Pull request by fluteds**
Hello! I've updated the Pull Request template to include the following:

- Markdown Formatting.
- Added guidelines.
- Checkbox for agreeing to give up recipes to the public domain & acknowledging that you've read the terms.
- Lists/examples of what to do and what not to do.

This PR is only a suggestion and I am more than happy to include ideas and change things around to suit needs. I'm hoping this is clearer for submitters to follow and understand as this can also be used as a 'written' agreement for wavering rights and reading guidelines.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+- File names should be the name of the dish with any spaced words separated by hyphens.
+  - ✅ `dish-name.md`
+  - ❌ `dish_name.md`
+  - ❌ `dish name.md`
+- Be sure to add a small number of relevant  tags to your recipe (and remove the dummy tags). Try to use already existing tags.
+  - ✅ `;tags: side american`
+  - ❌ `;tags: tag1 tag2 tag3`
+  - ❌ `;tags: tag1 tag2 tag3 side american`
+  - ❌ `;tags: side, american`
+
```
Comment (Knarkzel) [inline .github/pull_request_template.md:23]: Inconsistent spacing.

Comment (fluteds) [reply .github/pull_request_template.md:23]: @Knarkzel is the spacing in the tags? or elsewhere

Comment (fluteds) [reply .github/pull_request_template.md]: Resolved in latest push.

**Review by DGKSK8LIFE — APPROVED** [review]

**Comment by fluteds** [comment]
Closing due to inactivity.

**Pull request closed without merging**